receipt_dynamo/fix_exceptions_single.py

```
@@ -0,0 +1,238 @@
+
+
+def print_analysis(filepath: Path, results: Dict):
+    """Print analysis results."""
+    print(f"\nAnalysis for {filepath.name}")
+    print("=" * 60)
+
+    if not results["ValueError"] and not results["RuntimeError"]:
+        print("✅ No inconsistent exceptions found!")
+        return
+
+    print(f"\n📊 Summary:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)